diagnostic.md

```diff
@@ -48,11 +53,16 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    <!-- I'm unclear on how to answer but i think '/movies/:movie_id/123  -->
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    <!-- {{model.data}} I think
+    or like this
+    {{#each model as |data|}}
+      {{#link-to 'data' data tagName='li'}} {{data.name}} {{/link-to}}
+    {{/each}}
+     -->
```

Comment:
The key here is `model`. That hook allows us any and all of the access to that data. 1/2 credit.